src/main/java/com/dev/cinema/model/dto/MovieSessionResponseDto.java

```diff
@@ -1,11 +1,12 @@
 package com.dev.cinema.model.dto;
 
 public class MovieSessionResponseDto {
+    private Long id;
     private String movieTitle;
-    private int cinemaHallCapacity;
     private String localDateTime;
     private Long movieId;
     private Long cinemaHallId;
+    private Long movieSessionId;
```

Comment:
what is the difference between movieSessionId and id?

Reply:
exactly, it is the same